frequencyChar.js

```diff
@@ -0,0 +1,20 @@
+function itemFrequency(array) {
+    let characterFrequency = {};
+    let totalCount = 0
+  
+    for (let item of array) {
+        totalCount+= 1
+      if (!characterFrequency[item]) {
+        // if not in the dictionary
+        characterFrequency[item] = 1;
+      } else {
+        characterFrequency[item] += 1;
+      }
+    } for (let keys of Object.keys(characterFrequency)) {
+       characterFrequency[keys]= (characterFrequency[keys]/totalCount)*100 +' %';
+    
+    }
+    return characterFrequency;
+}
+console.log(itemFrequency(['h', 'e', 'y', 'e', 'e']));
```

Comment:
If you want to include this for testing purpose, use the `if(require.main === module) { ... }` pattern. Otherwise, this code will be run every time you use `require` to load it.